onnxruntime/core/session/inference_session.h

```diff
@@ -128,7 +135,16 @@ class InferenceSession {
     * Calling this API is optional.
     * @return OK if success.
     */
-  common::Status RegisterGraphTransformer(std::unique_ptr<onnxruntime::GraphTransformer> p_graph_transformer);
+  common::Status RegisterGraphTransformer(std::unique_ptr<onnxruntime::GraphTransformer> p_graph_transformer, 
```

Comment:
What are the semantics of this call? Is this for registering additional providers to L2 only?

Reply:
Yes right now to keep it simple we only register transformers to L2... There is no use case right now to do it otherwise... We can extend this if there such a requirement comes...

Reply:
Let's add a level and default it to L2. Please mention it in a comment.